subsys/bluetooth/mesh/adv.c

```diff
@@ -60,10 +64,11 @@ static struct bt_mesh_adv *adv_alloc(int id)
 	return &adv_pool[id];
 }
 
-struct net_buf *bt_mesh_adv_create_from_pool(struct net_buf_pool *pool,
-					     bt_mesh_adv_alloc_t get_id,
-					     enum bt_mesh_adv_type type,
-					     uint8_t xmit, k_timeout_t timeout)
+static struct net_buf *bt_mesh_adv_create_from_pool(struct net_buf_pool *pool,
+						    bt_mesh_adv_alloc_t get_id,
+						    enum bt_mesh_adv_type type,
+						    enum bt_mesh_adv_tag tag,
```

Comment:
@jhedberg `enum bt_mesh_adv_tag tag` only used `BT_MESH_LOCAL_ADV` and `BT_MESH_RELAY_ADV`, so 1 bit enough.